config/locales/sv.yml

```diff
@@ -700,6 +709,51 @@ sv:
       delete_confirm: "Är du säker på att du vill radera orsaken: %{name_sv} (%{name_en})?"
 
 
+  shf_mailer:
+    greeting: &email_greeting "Hej %{greeting_name}"
+
+    trouble_reading_browser: "Har du problem med att läsa detta? Visa det i en webbläsare"
+
+    email_us: "Om du har några frågor eller förslag, maila oss på "
+
+    footer:
+      text:
+        shf_name: *SHF_name
+        shf_motto: KOMPETENS – GLÄDJE – OMTANKE
+        email_sent_to: Det här mailet skickades till %{email_sent_to}
+        why_you_are_receiving: Du får detta mail för att du är medlem i SVERIGES HUNDFÖRETAGARE
+
+    admin:
+      signoff: Tack
+      signature: &medlemsansvarig Medlemsansvarig
+
+      new_application_received:
+        subject: Ny ansökan i systemet Hej
+        message_text:
+          new_app_arrived: "En ny ansökan har inkommit som du borde kika på. "
+          from: Från
+          view_app_here: "Du kan se medlemskapsansökan här: "
+          must_be_logged_in: "(Du måste vara inloggad som en administratör för att se det.)"
+
+
+    membership_application:
+      signoff: "Med vänliga hälsningar,"
+      signature: *medlemsansvarig
+      signature_url: "http://sverigeshundforetagare.se"
+
+      acknowledge_received:
+        subject: Din ansökan till Sveriges Hundföretagare är mottagen
+        message_text: "Din ansökan är mottagen, på grund av hög arbetsbelastning tar det en stund innan vi kommer till den. Så var inte orolig om det tar en stund innan du hör från oss."
+
+      accepted:
+        subject: "Välkommen som medlem i Sveriges Hundföretagare"
+        greeting: *email_greeting
+        message_text:
+          youre_active: "Du har nu aktiverats som medlem i systemet och kan logga in och hantera din profil och bygga upp din företagssida. När du är inloggad kan du redigerar din företagssida här: "
```

Comment:
*redigerar* should be *redigera* 

Link after this line should be to the edit page.